net-mgmt/zabbix-agent/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		zabbix-agent
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		Enterprise-class open source distributed monitoring agent
+PLUGIN_DEPENDS=     zabbix32-agent
```

Comment:
Maybe also add a upgrade warning to the release notes, just in case someone is already using Zabbix Agent on OPNsense without a GUI.